src/3_PB/PianoDiQualifica/content.typ

```diff
@@ -924,18 +1249,115 @@ Tale strumento ha mostrato immediatamente i suoi effetti benefici: gli errori ne
+
+Si rimanda al documento _Specifiche Tecniche v1.0_ per vedere la lista completa dei requisiti e il relativo soddisfacimento.
 
+=== Manutenibilità
+
+#figure(
+  image("./assets/valutazione-metriche/FANIN_FANOUT.png", width: 85%),
+  caption: [
+    Valutazione metrica FAN-IN FAN-OUT.
+  ],
+)
+
+FAN-IN rappresenta il numero di moduli o componenti direttamente collegati o dipendenti da un modulo o una funzione specifica. 
+SFOUT rappresenta il numero di dipendenze o connessioni che un componente o modulo particolare ha con altri componenti o moduli. Misura quanti altri elementi dipendono o interagiscono con un dato elemento all'interno di un #glossary[sistema].
+
+L'analisi dei valori numerici evidenzia che il fan-in si attesta intorno al valore 2,6, indicando che ogni modulo o componente specifico ha in media circa 2,6 altre parti del #glossary[sistema] che dipendono direttamente da esso. Questo valore suggerisce che, sebbene ci sia un certo grado di dipendenza da altre parti del #glossary[sistema], questo non è così elevato da indicare una massima utilizzazione del modulo da altre parti del #glossary[sistema].
+
+Per quanto riguarda il fan-out, che si attesta intorno al valore di 5,62, questo indica che ogni modulo ha in media circa 5,62 dipendenze o connessioni con altri moduli o componenti all'interno del #glossary[sistema]. Un valore così elevato di fan-out potrebbe suggerire che alcuni moduli hanno molte dipendenze da altri moduli, il che potrebbe aumentare la complessità del #glossary[sistema] e rendere più difficile la gestione dei moduli stessi.
+
+
+
+
+=== Usabilità
+
+#figure(
+  image("./assets/valutazione-metriche/FU.png", width: 85%),
+  caption: [
+    Valutazione facilità di utilizzo.
+  ],
+)
+
+Durante il corso degli #glossary[sprint], il numero massimo di click necessari per accedere a tutte le funzionalità del prodotto è rimasto costante a cinque. Questo rappresenta il valore ideale, confermando l'efficacia dell'interfaccia utente nel garantire un accesso rapido e diretto alle varie funzionalità del prodotto.
+
+#figure(
+  image("./assets/valutazione-metriche/TA.png", width: 85%),
+  caption: [
+    Valutazione tempo di apprendimento.
+  ],
+)
+
+Per quanto riguarda la metrica del tempo di apprendimento (TA), è stato condotto un test regolare da parte di una persona estranea al progetto una volta a settimana al fine di valutare la facilità d'uso del prodotto. Inizialmente, durante il primo test di familiarizzazione, è stato registrato un tempo di apprendimento di circa dieci minuti. Questo periodo è stato considerato necessario per consentire alla persona di prendere confidenza con l'interfaccia e le funzionalità del prodotto.
```

Comment:
se ci chiede chi è questa persona cosa facciamo?
tralaltro non è verosimile che il tempo di apprendimento cambi dato che alla fine stiamo utilizzando grafana che rimane di base quello.

Reply:
più che altro è impossibile da calcolare oggettivamente questa metrica. 
cmq si grafana rimane quello ma metti la velocità dell'utilizzatore migliora di volta in volta sapendo dove deve parare

Reply:
si ma il test dovrebbe essere a priori dell'utilizzatore, cioè considero siano ogni volta la stessa persona che però ha perso la memoria, almeno io la interpreto cos'

Reply:
si

Reply:
si ma quello che dico è che la metrica dovrebbe indicare la complessità della prodotto, non quanto ci si familiarizza nel tempo

Reply:
faccio la tabella con una riga a sto punto